cli/azd/pkg/graphsdk/request_info.go

```diff
@@ -0,0 +1 @@
+package graphsdk
```

Comment:
Do we need this file?

Reply:
Nope, just an empty file i forgot to delete :)